### PR TITLE
[BTS-153] Card Component

### PR DESCRIPTION
**Pull request by allyssonsantos**
card

### CODE REVIEW

File: components/Card/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  Thumbnail,
+  Content,
+  Footer,
+} from './sub-components';
+
+const CardWrapper = styled.article`
+  background-color: ${Colors.WHITE};
+  border-radius: 8px;
+  box-shadow: 0 2px 6px 0 ${Colors.SHADOW[50]};
+  color: ${Colors.BLACK[700]};
```
Comment (alan-oliv) [inline components/Card/Card.jsx]: I guess this color should inherit from body according to @ddsilva, right?

Comment (allyssonsantos) [reply components/Card/Card.jsx]: Right!
Changed.

File: stories/Card/Card.story.jsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                  Footer is provided for include actions buttons, icons, any
+                  kind of user interaction.
+                </td>
+              </tr>
+            </tbody>
+          </Table>
+
+          <p>
+            <code>
+              {'<Card.Title />'}, {'<Card.Description />'} and{' '}
```
Comment (ggdaltoso) [inline stories/Card/Card.story.jsx]:
```suggestion
              {`<Card.Title />, <Card.Description /> and <Card.Thumbnail />`}
```
💅

File: stories/Card/Card.story.jsx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+import SimpleCard from './examples/SimpleCard';
+import JobAdCard from './examples/JobAdCard';
+import AdvancedCard from './examples/AdvancedCard';
+
+import { Container, Col, Row } from '../../components/Grid';
+import Card from '../../components/Card';
+import { Typography } from '../../components/GlobalStyle';
+
+const importCard = `import { Card } from '@catho/quantum';`;
```
[the remark this answers is not in the document]
Comment (allyssonsantos) [reply stories/Card/Card.story.jsx]: Okay.

File: stories/Card/Card.story.jsx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    <Heading name="Card">
+      Cards contain content and actions about a single subject.
+    </Heading>
+    <TabbedView>
+      <Tab title="Usage">
+        <StoryContainer>
+          <Title as="h2">Importing Card</Title>
+          <SimpleHighlight>{importCard}</SimpleHighlight>
+
+          <Title as="h3">Compund Components</Title>
```
[the remark this answers is not in the document]
Comment (allyssonsantos) [reply stories/Card/Card.story.jsx]: Fixed.

File: stories/Card/Card.story.jsx (hunk 2 above)
Comment (allyssonsantos) [reply stories/Card/Card.story.jsx]: Changed.

**Pull request merged**